Dockerfile.geth

```diff
@@ -0,0 +1,71 @@
+ARG NODE_VERSION=16.14.0
+ARG ALPINE_VERSION=3.16
+ARG PYTHON_VERSION=3.10.4
+
+FROM node:${NODE_VERSION}-alpine AS node
+
+FROM python:${PYTHON_VERSION}-alpine${ALPINE_VERSION}
+
+COPY --from=node /usr/lib /usr/lib
+COPY --from=node /usr/local/share /usr/local/share
+COPY --from=node /usr/local/lib /usr/local/lib
+COPY --from=node /usr/local/include /usr/local/include
+COPY --from=node /usr/local/bin /usr/local/bin
+
+ENV GETH_DATA_DIR=/ethereum/data/
+ENV GETH_ETHEREUM_ACCOUNT=0x944466f81e70c74c0f6b736ba72c7a69e475735a
+ENV KEEP_ETHEREUM_PASSWORD=password
+ENV NETWORK=development
+
+RUN apk update 
+RUN apk add --update --no-cache bash git geth make gcc g++ && \
+  npm install --global yarn --force
+
+# There's a transitive git problem that this fixes
+RUN git config --global url."https://".insteadOf git://
+
+COPY ./solidity-contracts /solidity-contracts
+WORKDIR /solidity-contracts
+RUN git init
```

Comment:
i forget which repository needs to register as a git repo for the later commands to work, but it's at least one of them

having the projects as submodules means that rather than having a `.git/` folder, they have `.git` files that point back at the parent directory, and that parent directory doesn't exist when we're copying

I can definitely go back and see which ones we can safely remove!